exec/java-exec/src/main/java/org/apache/drill/exec/expr/ClassGenerator.java

```diff
@@ -218,11 +220,16 @@ public HoldingContainer addExpr(LogicalExpression ex) {
   }
 
   public HoldingContainer addExpr(LogicalExpression ex, boolean rotate) {
+    return addExpr(ex, rotate, false);
```

Comment:
I think this had better to be addExpr(ex, rotate, true). That is, be default, we turn on CSE. Otherwise, seems the patch only turns on CSE for Project, Filter, HashAgg, HashJoin, but not for other operators. 

Is there any reason that we do not by default turn on CSE here?